From 5b7b1f80cbaa88849e2698d67bf2d72ac9addac4 Mon Sep 17 00:00:00 2001
From: Breno Leitao <breno.leitao@gmail.com>
Date: Wed, 5 Apr 2017 13:28:07 +0000
Subject: [PATCH] abuild: Add support for lzip source packages

Currently abuild does not understand .tar.lz packages, which blocks
the inclusion of certain packages in Alpine Linux.

I found this issue when adding 'ed' package to the repository. With
this change, abuild package will now depend on lzip package. I might
send a patch for abuild's APKBUILD.
---
 abuild.in | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/abuild.in b/abuild.in
index c201eb2..843baab 100644
--- a/abuild.in
+++ b/abuild.in
@@ -418,6 +418,9 @@ default_unpack() {
 			*.tar.bz2)
 				msg "Unpacking $s..."
 				tar -C "$srcdir" -jxf "$s" || return 1;;
+			*.tar.lz)
+				msg "Unpacking $s..."
+				tar -C "$srcdir" --lzip -xf "$s" || return 1;;
 			*.tar.lzma)
 				msg "Unpacking $s..."
 				unlzma -c "$s" | tar -C "$srcdir" -x  \
-- 
2.12.2

